sources/procfs.go

```diff
@@ -134,68 +134,68 @@ func newLustreProcMetric(filename string, promName string, source string, path s
-			{"job_stats", "job_write_minimum", writeMinimumHelp, s.counterMetric},
-			{"job_stats", "job_write_maximum", writeMaximumHelp, s.counterMetric},
-			{"job_stats", "job_write_total", writeTotalHelp, s.counterMetric},
-			{"job_stats", "job_num_getattr", getattrHelp, s.counterMetric},
-			{"job_stats", "job_num_setattr", setattrHelp, s.counterMetric},
-			{"job_stats", "job_num_punch", punchHelp, s.counterMetric},
-			{"job_stats", "job_num_sync", syncHelp, s.counterMetric},
-			{"job_stats", "job_num_destroy", destroyHelp, s.counterMetric},
-			{"job_stats", "job_num_create", createHelp, s.counterMetric},
-			{"job_stats", "job_num_statfs", statfsHelp, s.counterMetric},
-			{"job_stats", "job_num_get_info", getInfoHelp, s.counterMetric},
-			{"job_stats", "job_num_set_info", setInfoHelp, s.counterMetric},
-			{"job_stats", "job_num_quotactl", quotactlHelp, s.counterMetric},
-			{"kbytesavail", "kbytesavail", "Number of kilobytes readily available in the pool", s.counterMetric},
-			{"kbytesfree", "kbytesfree", "Number of kilobytes allocated to the pool", s.counterMetric},
-			{"kbytestotal", "kbytestotal", "Capacity of the pool in kilobytes", s.counterMetric},
-			{"lfsck_speed_limit", "lfsck_speed_limit", "Maximum operations per second LFSCK (Lustre filesystem verification) can run", s.counterMetric},
-			{"num_exports", "num_exports", "Total number of times the pool has been exported", s.counterMetric},
-			{"precreate_batch", "precreate_batch", "Maximum number of objects that can be included in a single transaction", s.counterMetric},
-			{"recovery_time_hard", "recovery_time_hard", "Maximum timeout 'recover_time_soft' can increment to for a single server", s.counterMetric},
-			{"recovery_time_soft", "recovery_time_soft", "Duration in seconds for a client to attempt to reconnect after a crash (automatically incremented if servers are still in an error state)", s.counterMetric},
-			{"soft_sync_limit", "soft_sync_limit", "Number of RPCs necessary before triggering a sync", s.counterMetric},
+			{"blocksize", "blocksize_bytes", "Filesystem block size in bytes", s.gaugeMetric},
+			{"brw_size", "brw_size_megabytes", "Block read/write size in megabytes", s.gaugeMetric},
+			{"brw_stats", "pages_per_bulk_rw_total", pagesPerBlockRWHelp, s.counterMetric},
+			{"brw_stats", "discontiguous_pages_total", discontiguousPagesHelp, s.counterMetric},
+			{"brw_stats", "disk_io_now", diskIOsInFlightHelp, s.gaugeMetric},
+			{"brw_stats", "io_time_milliseconds_total", ioTimeHelp, s.counterMetric},
+			{"brw_stats", "disk_io_total", diskIOSizeHelp, s.counterMetric},
+			{"degraded", "degraded_now", "Binary indicator as to whether or not the pool is degraded - 0 for not degraded, 1 for degraded", s.gaugeMetric},
+			{"filesfree", "files_free_now", "The number of inodes (objects) available", s.gaugeMetric},
+			{"filestotal", "file_capacity", "The maximum number of inodes (objects) the filesystem can hold", s.counterMetric},
```

Comment:
Looks like this one was double-confirmed by @knweiss running promtool check-metrics. I'll update it!